roles/os_firewall/tasks/iptables.yml

```diff
@@ -4,13 +4,17 @@
   systemd:
     name: firewalld
     state: stopped
-    enabled: no
-    masked: yes
   register: task_result
   failed_when:
     - task_result is failed
     - ('could not' not in task_result.msg|lower)
 
+- name: Mask and disable firewalld service
```

Comment:
Let's add a comment here for reasoning, otherwise looks good.

Reply:
Done